src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/cocoapods-installation.mdx

```diff
@@ -77,9 +77,17 @@ These procedures to configure your iOS/tvOS app with CocoaPods and Objective-C a
     if [ ${CONFIGURATION} = "Debug" ]; then
         echo "Skipping DSYM upload CONFIGURATION: ${CONFIGURATION}"
         exit 0
-    fi
+    fi 
```

Comment:
@cdillard-NewRelic Is this a typo or is it fine?

Reply:
its fine, in shell scripts if statements look this way.